libtock/sensors/rainfall.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "../tock.h"
+#include "syscalls/rainfall_syscalls.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Function signature for rainfall data callback.
+//
+// - `arg1` (`int`): Returncode indicating status from sampling the sensor.
+// - `arg2` (`int`): rainfall in hundredths of percent.
```

Comment:
It seems like the description for `arg2` should match the description of the `rainfall` argument in `libtock-sync/sensors/rainfall.h`?